docs/metasploit-framework.wiki/Metasploit-Guide-Session-Search.md

```diff
@@ -0,0 +1,60 @@
+## Session Search
+
+When you have a number of sessions open, searching can be a useful tool to navigate them. This guide explains what capabilities are available for navigating open sessions with search.
+
+You can get a list of sessions matching a specific criteria within msfconsole:
+
+```msf
+msf6 payload(windows/meterpreter/reverse_http) > sessions --search "session_id:1 session_id:2"
+Active sessions
+===============
+
+  Id  Name  Type            Information      Connection
+  --  ----  ----            -----------      ----------
+  1         meterpreter x8  WIN-ED9KFH65RDH  192.168.2.1:444
```

Comment:
We try to avoid giving doc examples that have word wrapping; on the browser they'll be scrolled for the user automagically so there's no need for us to have the new lines here :+1: